backends-common/jpa/src/main/java/org/apache/james/backends/jpa/TransactionRunner.java

```diff
@@ -76,8 +76,8 @@ public void run(Consumer<EntityManager> runnable) {
         }
```

Comment:
> JAMES-3179 Use a Consumer instead of a Function in TransactionRunner 

Why?

(Whithout an explanation I might do the mistake again)

Reply:
we don't use the return value?

Reply:
In fact, we require a `Function` but we ignore the result, making the interface suspicious